index.html

```diff
@@ -50,124 +54,170 @@ <h1>Im Lucca
+                    <button type="button">javascript</button>
+                </div> 
+                <div class="buttonSee">
+                    <button type="button">See Project</button>
+                </div>  
             </div>
-            <div class="Buttons">
-                <button type="button">html</button>
-                <button type="button">css</button>
-                <button type="button">javascript</button>
-            </div> 
-            <div class="buttonSee">
-                <button type="button">See Project</button>
-            </div>  
         </div>
     </section>
     
     <section class="all5">
-        <div class="document5">
-            <h2 class="about"> About Myself </h2>
-            <img class="sub1" src="images/PrimaryText.jpg" alt="">
-            <h3 class="lets">LETS CONNECT</h3>
-        </div>
-        <div class="part5">
-            <div class="flex">
-                <ul class="point6">
-                    <li> <a href="https://github.com/luccamicale"><img class="git1" src="images/Vectorgit.jpg" alt=""></a></li>
-                    <li> <a href="#"><img class="letteri1" src="images/VectorI.png" alt=""></a></li>
-                    <li> <a href="#"><img class="lettern1" src="images/Vectorn.jpg" alt=""></a></li>
-                    <li> <a href="#"><img class="letterm1" src="images/Vectorm.jpg" alt=""></a></li>
-                    <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter1" src="images/Vectortw.jpg" alt=""></a></li>
-                    <li> <a href=""><img class="hand1" src="images/Vectorhnad.jpg" alt=""></a></li>
-                </ul>
+        <div class="about-sub-1">
+            <div class="document5">
+                <h2 class="about"> About Myself </h2>
+                <h3 class="sub1"> Hello Im a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded,
+                      dont hestiate to contact me.</h3>
+                <h3 class="lets">LETS CONNECT</h3>
+            </div>
+            <div class="part5">
+                <div class="flex">
+                    <ul class="point6">
+                        <li> <a href="https://github.com/luccamicale"><img class="git1" src="images/Vectorgit.jpg" alt=""></a></li>
+                        <li> <a href="#"><img class="letteri1" src="images/VectorI.png" alt=""></a></li>
+                        <li> <a href="#"><img class="lettern1" src="images/Vectorn.jpg" alt=""></a></li>
+                        <li> <a href="#"><img class="letterm1" src="images/Vectorm.jpg" alt=""></a></li>
+                        <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter1" src="images/Vectortw.jpg" alt=""></a></li>
+                        <li> <a href=""><img class="hand1" src="images/Vectorhnad.jpg" alt=""></a></li>
```

Comment:
- [ ] Your portfolio is looking great! Let's make it even better by fixing the bug about the "About myself" and "Contact" sections:

|1400px|1200px|900px|729px|
|---|---|---|---|
|![Screen Shot 2022-09-02 at 3 24 58 PM](https://user-images.githubusercontent.com/83717285/188231197-824c895b-cacd-403c-ad79-4e9c9a9ff238.png)|![Screen Shot 2022-09-02 at 3 25 19 PM](https://user-images.githubusercontent.com/83717285/188231229-f8596f1c-8f7a-44c2-8fc0-f29cdb2dbbc7.png)|![Screen Shot 2022-09-02 at 3 25 46 PM](https://user-images.githubusercontent.com/83717285/188231247-7832bb0e-2771-4de2-82ab-5772c46e76af.png)|![Screen Shot 2022-09-02 at 3 26 02 PM](https://user-images.githubusercontent.com/83717285/188231260-34dc4fa3-b428-46e9-b6f7-446228ff68b2.png)|

- [ ]  The About section's columns are overlapping when the width is under around 1200px, it would be great to fix that.
- [ ]  The contact form's background has a blank space on the right side when the width is under 1400px, , it would be great to fix this as well.